test/SettingsMigrator.UnitTests/SettingsMigrator.UnitTests.csproj

```diff
@@ -7,7 +7,8 @@
   </PropertyGroup>
   <Import Project="$(TestPlatformRoot)scripts/build/TestPlatform.Settings.targets" />
   <PropertyGroup>
-    <TargetFramework>net451</TargetFramework>
+    <!-- TODO: Do we need this for net6.0 as well? -->
```

Comment:
If that's not adding too much delay, might be worth to ensure that's also working with net6 IMO.

Reply:
I don't want to mix in more changes.